site/content/docs/5.0/utilities/borders.md

```diff
@@ -43,6 +43,16 @@ Change the border color using utilities built on our theme colors.
 <span class="border border-white"></span>
 {{< /example >}}
 
+### Border-width
+
+{{< example >}}
```

Comment:
@MauricioHernanCabrera, you missed this class here (this used to be ok):
```suggestion
{{< example class="bd-example-border-utils" >}}
```

Reply:
Ah, yeah I accidentally removed that yesterday